src/layout/MarkdownFormatter.jsx

```diff
@@ -370,38 +370,40 @@ export function MarkdownFormatter({ markdown, backgroundColor, dict }) {
           }
         },
         h2: ({ children }) => {
-          let headerText = children[0];
+          let headerText = React.Children.toArray(children).join("");
```

Comment:
Thanks for your work on this, Tawanda. While this may work, there are a lot of reasons why you probably wouldn't want to use `React.Children` in this code; you can read more about why at https://react.dev/reference/react/Children. In a nutshell, it can make the code very fragile, so we would ask that you replace this implementation with a different one.

Reply:
In fact, a great reason why `React.Children` could be problematic here is if we intend to use `MarkdownFormatter` in a broader way, to format any markdown we might want to display. Doing so would keep our styling consistent and make it easier to build out new markdown-driven pages.